masterlist.yaml

```
@@ -12084,15 +12084,18 @@ plugins:
           - 'YOT - Your Own Thoughts SE - First Person Messages'
           - '[YOT - Your Own Thoughts SE - First Person Messages](https://www.nexusmods.com/skyrimspecialedition/mods/762/)'
         condition: 'active("YOT - Your Own Thoughts.esp") and not active("YOT - CWRepairs.esp")'
-      - <<: *patchProvided
+      - <<: *patchIncluded
         subs: [ 'Immersive Citizens - AI Overhaul' ]
         condition: 'active("Immersive Citizens - AI Overhaul.esp") and not active("CWR_IC_Patch.esp")'
-      - <<: *patchProvided
+      - <<: *patchIncluded
         subs: [ 'Open Cities Skyrim' ]
         condition: 'active("Open Cities Skyrim.esp") and not active("CWR_OC_Patch.esp")'
-      - <<: *patchProvided
+      - <<: *patchIncluded
         subs: [ 'The Second Great War' ]
         condition: 'active("GreatWarSkyrim.esp") and not active("CWRepairSGW.esp")'
+      - <<: *patchProvided
+        subs: [ 'JK''s Temple of the Divines' ]
+        condition: 'active("JK''s Temple of the Divines.esp") and not active("(CWR_JKTemple_Patch|JKs Temple of the Divines - After the Civil War patch)\.esp")'
```

The regex doesn't simplify much here, & the patches are from separate pages anyway.

```suggestion
        condition: 'active("JK''s Temple of the Divines.esp") and not (active("CWR_JKTemple_Patch.esp") or active("JKs Temple of the Divines - After the Civil War patch.esp"))'
```